README.md

```diff
@@ -437,8 +437,10 @@ If you run Ubuntu 19.04 (Disco Dingo) or newer, you can install the
 sudo apt install fd-find
 ```
 Note that the binary is called `fdfind` as the binary name `fd` is already used by another package.
-It is recommended that you add an `alias fd=fdfind` to your shells initialization file, in order to
-use `fd` in the same way as in this documentation.
+It is recommended that after installation, you add a link to `fd` by executing command
+`ln -s $(which fdfind) ~/.local/bin/fd`, in order to use `fd` in the same way as in this documentation.
+You can also add `alias fd=fdfind` to your shells initialization file but this wouldn't help if you
+export variables using `fd`. You will have to use `fdfind` in export also.
```

Comment:
I think we can even remove this part. Also below. Let's rather try to keep the install instructions short.